docs/src/views/Colors.tsx

```diff
@@ -0,0 +1,352 @@
+                  that actual token's value is dependent on theme.
+                </b>{' '}
+                You can see an example of tokens defined for Teams theme{' '}
+                <Text as={Link} weight="bold" content="here" color="brand" to="color-schemes" />.
+              </li>
+            </ul>
+          </p>
+          <p>
+            The color tokens defined in the color scheme are mapped to actual values for all themes
+            used in the application. This means that, if the developers use some token from the
+            schema, it will be mapped to the correct color value provided by the current theme.
+          </p>
+
+          <p>Let's illustrate this with code example:</p>
+
+          <ExampleSnippet
+            value={`
+            import React from 'react'
+            import { Button } from '@stardust-ui/react'
+
+            const theme = {
+              componentStyles: {
+                Box: {
+                  root: ({ theme: { siteVariables } }) => {
+                    const colorScheme = siteVariables.colorScheme
+                    return {
+                      color: colorScheme.brand.foreground3,
+                      backgroundColor: colorScheme.brand.background,
+                      border: \`1px solid \${colorScheme.brand.border}\`,
+                      ':hover': {
+                        color: colorScheme.brand.foregroundHover,
+                        backgroundColor: colorScheme.brand.backgroundHover,
+                        border: \`1px solid \${colorScheme.brand.borderHover}\`,
+                      },
+                    }
+                  },
+                },
+              },
+            }
+
+            const ColorSchemeExample = () => (
+              <Provider theme={mergeThemes(themes.teams, theme)}>
+                 <Text content={'Box in light theme'} />
+                 <Box content={'LIGHT THEME - HOVER ME'}/>
+                 <Text content={'Box in high contrast theme'} />
+                 <Provider theme={themes.teamsHighContrast}>
+                   <Box content={'HIGH CONTRAST THEME - HOVER ME'}/>
+                 </Provider>
+              </Provider>
+            )
+
+            export default ColorSchemeExample
+        `}
+            render={() => (
+              <Provider theme={mergeThemes(themes.teams, theme)}>
+                <Text content={'Box in light theme'} />
+                <Box content={'LIGHT THEME - HOVER ME'} />
+                <Text content={'Box in high contrast theme'} />
+                <Provider theme={themes.teamsHighContrast}>
+                  <Box content={'HIGH CONTRAST THEME - HOVER ME'} />
+                </Provider>
+              </Provider>
+            )}
+          />
+
+          <p>
+            Here is how one color scheme looks like, defined for brand in Teams light, high contrast
+            and dark themes.
+          </p>
+
+          <Fader url={'color-schemes'}>
+            <ColorSchemes
```

Comment:
![image](https://user-images.githubusercontent.com/14183168/59514689-a766b500-8ebd-11e9-8353-e81d7201fbf4.png)

Not sure, but it seems that this block should be last

Reply:
Not sure I understand the question. Is there text where?

Reply:
Meaning the text is not vertically aligned. It looks like there is some space under each color text, maybe because of the first column where the text goes on 2 lines.